x/ecocredit/expected_keepers.go

```diff
@@ -42,5 +42,11 @@ type DistributionKeeper interface {
 }
 
 type ParamKeeper interface {
+
+	// Get fetches a parameter by key from the Subspace's KVStore and sets the provided pointer to the fetched value.
+	// If the value does not exist, this method will panic.
+	Get(ctx sdk.Context, key []byte, ptr interface{})
```

Comment:
this is an SDK method, I can't change the implementation or the signature. To my understanding, panicking isn't really an unsafe operation, just a code smell. the SDK has a global panic handler so its mostly just an ugly way to return an error from a module method.

Reply:
i could wrap our param calls in a custom `GetParam` function that has its own panic recovery mechanism, but not sure how valuable that is given this will all go away next upgrade

Reply:
Yea, I guess its not necessarily not safe and we would always catch the panic before upgrading a live network and the parameter would never change, so a panic would be ok.

Reply:
I don't think we need our own panic recovery mechanism. I am wondering about whether we really need this now given that this will all go away in the next upgrade but it doesn't hurt, just a bit more work now.

Reply:
I don't think we need our own panic recovery mechanism. I'm wondering about whether we really need this given that this will all go away when we update to ORM params but it does improve performance in the short-term.